src/GitReleaseNotes.Website/Models/Api/Extensions/ReleaseNotesRequestExtensions.cs

```diff
@@ -7,37 +9,37 @@ public static Context ToContext(this ReleaseNotesRequest releaseNotesRequest)
             IIssueTrackerContext issueTrackerContext = null;
 
             var lowercaseUrl = releaseNotesRequest.IssueTrackerUrl.ToLower();
-            if (lowercaseUrl.Contains("bitbucket"))
-            {
-                issueTrackerContext = new BitBucketContext
-                {
-                    Url = releaseNotesRequest.IssueTrackerUrl
-                };
-            }
+            //if (lowercaseUrl.Contains("bitbucket"))
```

Comment:
It would be good to try and get two working at the same time, otherwise the design changes might get painful to add the second one. Understand that might be too much though